snappass/templates/set_password.html

```diff
@@ -3,27 +3,41 @@
 {% block content %}
 <div class="container">
   <section>
-    <div class="page-header"><h1>Password Share</h1></div>
-    <form class="form-horizontal" id="password_create" method="post">
-      <input type="text" id="password" name="password" autofocus="True" placeholder="Share a Password" autocomplete="off">
-      <select name="ttl">
-        <option>Week</option>
-        <option>Day</option>
-        <option>Hour</option>
-      </select>
-      <button type="submit" class="btn btn-primary" id="submit">Generate url</button>
-    </form>
-    <h3>Using this tool</h3>
-      <p>This tool allows you to share passwords in a secure way.
-      <ol>
-          <li>Enter the password and its expiration time</li>
-          <li>Share the one time use link</li>
-      </ol>
-      </p>
-      <p>
-      Note that the link will <i>only work once</i>.
-      </p>
-  </section>
+    <div class="page-header"><h1>Set Password</h1></div>
+    <div class="row">
+      <form role="form" id="password_create" method="post">
+        <div class="col-sm-6 margin-bottom-10">
+          <div class="input-group">
+            <span class="input-group-addon" id="basic-addon1"><span class="glyphicon glyphicon-lock" aria-hidden="true"></span></span>
+            <input type="text" id="password" name="password" class="form-control" placeholder="Password" aria-describedby="basic-addon1" autofocus>
+          </div>
+        </div>
+
+        <div class="col-sm-2 margin-bottom-10">
+          <select class="form-control" name="ttl">
+            <option value="Week">Week</option>
+            <option value="Day">Day</option>
+            <option value="Hour">Hour</option>
+          </select>
+        </div>
+
+        <div class="col-sm-4">
+          <button type="submit" class="btn btn-primary" id="submit">Generate URL</button>
+        </div>
+      </form>
+    </div>
+
+    <h3>How SnapPass Works</h3>
+    <p>SnapPass allows you to share passwords with anybody in a secure, ephemeral way.</p>
```

Comment:
I prefer the original copy, mostly because it's shorter and more to the point. Can we revert it?